Update helm chart examples (#78)
jupyterhub/ltiauthenticator: CONTRIBUTING.md

  BREAKING CHANGE: `extends` key in config file is now used for extending other config files
  ```

#### Releases

> Please refer to the documentation within JupyterHub's [RELEASE.md](https://github.com/jupyterhub/jupyterhub/blob/main/RELEASE.md). We recomment using [`bump2version`](https://github.com/c4urself/bump2version) instead of `tbump`, however. The example below updates the release to `v1.3.0` and outputs the results (`--verbose`):

Reviewer: ```suggestion
Please refer to the documentation within JupyterHub's [RELEASE.md](https://github.com/jupyterhub/jupyterhub/blob/main/RELEASE.md). We recomment using [`bump2version`](https://github.com/c4urself/bump2version) instead of `tbump`, however. The example below updates the release to version `1.3.0` and outputs the results (`--verbose`):
```

```bash

> bump2version --verbose --new-version 1.3.0 minor

Reviewer: ```suggestion
bump2version --verbose --new-version 1.3.0 -
```

Reviewer: When a new version is explicitly defined, the last past is just a distraction that can be handled with `-`.

#### Releases

> Please refer to the documentation within JupyterHub's [RELEASE.md](https://github.com/jupyterhub/jupyterhub/blob/main/RELEASE.md). We recomment using [`bump2version`](https://github.com/c4urself/bump2version) instead of `tbump`, however. The example below updates the release to `1.3.0` and outputs the results (`--verbose`):

Reviewer: ```suggestion
Please refer to the documentation within JupyterHub's [RELEASE.md](https://github.com/jupyterhub/jupyterhub/blob/main/RELEASE.md). We recommend using [`bump2version`](https://github.com/c4urself/bump2version) instead of `tbump`, however. The example below updates the release to `1.3.0` and outputs the results (`--verbose`):
```

Reviewer: Don't we want to reference the local RELEASE.md file, at https://github.com/jupyterhub/ltiauthenticator/blob/main/RELEASE.md ?

Author: We do not! Thanks for the heads up.

Author: @consideRatio got my wires crossed there....all good now.

#### Releases

Please refer to the documentation within this repo's [RELEASE.md](https://github.com/jupyterhub/ltiauthenticator/blob/main/RELEASE.md). The example below updates the release to `1.3.0` and outputs the results (`--verbose`):

```bash
bump2version --verbose --new-version 1.3.0 -
```

> Use the `--dry-run` flag to emulate the update without actually changing the files.

Reviewer: What do you think about removing this besides the reference to RELEASE.md, and letting that document contain everything needed?

I'm worried that this example isn't helpful by itself, and/or the --dry-run flag idea is not part of RELEASE.md.

Author: Yeah, that makes sense, I'll remove it.
